src/main/webapp/includes/download-box-smack.jspf

```diff
@@ -30,7 +30,7 @@
 
         <div class="ignite_downloads_content">
             <span class="ignite_downloads_info">
-                <!-- <a href="#"><strong>Plugins</strong></a> |  -->
+                <a href="https://github.com/igniterealtime/Smack/releases"><strong>Old versions</strong></a> |
```

Comment:
I don't think smack needs this addition as old releases and even releases in general are handled by maven repos.

Reply:
Ok, i hive commented it out